--- Makeconf.orig 2016-09-20 16:33:23.061756020 +0200
+++ Makeconf  2016-09-20 16:39:47.385547117 +0200
@@ -1,26 +1,26 @@
 # set this to your compiler's executable name (e.g. gfortran, g77)
 FC=gfortran
 # requested flags
-FFLAGS=-fimplicit-none -O3 -funroll-loops 
+FFLAGS=-fimplicit-none -march=native -O3 -funroll-loops -lgfortran
 # set if you need shared library
 FPICFLAGS=-fPIC
 
 # BLAS library (only required for tests)
-BLAS=-lblas
+BLAS="/Users/jgoldfar/Public/octave/usr/lib/libopenblas.a"
 # LAPACK library (only required for tests)
-LAPACK=-llapack
+LAPACK=
 
 # Library version
 VERSION=1.1
 MAJOR=1
 
 # The default library dir
-LIBDIR=lib
+LIBDIR=lib64
 
 # Destination installation offset
 DESTDIR=
 
 # set default prefix to /usr/local
 ifeq ($(strip $(PREFIX)),)
-  PREFIX=/usr/local
+  PREFIX=/usr/local
 endif
